root/etc/cont-init.d/40-plex-first-run

```diff
@@ -67,10 +67,10 @@ fi
 if [ "${CHANGE_CONFIG_DIR_OWNERSHIP,,}" = "true" ]; then
   if [ -f "${prefFile}" ]; then
     if [ ! "$(stat -c %u "${prefFile}")" = "$(id -u plex)" ]; then
-      chown -R plex:plex /config
+      find /config \! \( -uid $(id -u plex) -gid $(id -g plex) \) -exec chown plex:plex {} \;
```

Comment:
Likely better to use `-print0` and pipe it into `xargs -0` as that's more efficient than `-exec`.